python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -316,23 +322,22 @@ def _concat_treelite_models(self):
         the treelite format and then concatenate the different treelite models
         to create a single model. The concatenated model is then converted to
         bytes format.
-
         """
-
         mod_bytes = []
+        model_protobuf_futures = list()
         for w in self.workers:
-            mod_bytes.append(self.rfs[w].result().model_pbuf_bytes)
-
+            model_protobuf_futures .append(
+                dask.delayed(RandomForestClassifier._get_protobuf_bytes)
+                (self.rfs[w]))
+        mod_bytes = self.client.compute(model_protobuf_futures, sync=True)
         last_worker = w
-
         all_tl_mod_handles = []
         model = self.rfs[last_worker].result()
         for n in range(len(self.workers)):
             all_tl_mod_handles.append(model._tl_model_handles(mod_bytes[n]))
 
-        concat_model_handle = model._concatenate_treelite_handle(
+        model._concatenate_treelite_handle(
             treelite_handle=all_tl_mod_handles)
-        model._concatenate_model_bytes(concat_model_handle)
```

Comment:
Do we still need this if we are not keeping the result anywhere? Or should this be stored somewhere?

Reply:
this function updates the self variable for `model` and then `model` is used to set the `self.local_model`. The `self.local_model` is the RF model used for performing lazy prediction.

Reply:
Ah ok. Thanks for clarifying.